pom.xml

```diff
@@ -1430,4 +1430,8 @@
       </plugin>
     </plugins>
   </reporting>
+
+  <prerequisites>
+    <maven>3.2</maven>
+  </prerequisites>
```

Comment:
Okay. Would you mind adding a comment in both places to change the other if making changes? :)